12-offer-encoding.md

```diff
@@ -706,23 +693,19 @@ Note that the recipient of the invoice can determine the expected
 amount from either the offer it received, or the invoice_request it
 sent, so often already has authorization for the expected amount.
 
-The default `relative_expiry` of 7200 seconds, which is generally a
+The default `invoice_relative_expiry` of 7200 seconds, which is generally a
 sufficient time for payment, even if new channels need to be opened.
 
 Blinded paths provide an equivalent to `payment_secret` and `payment_metadata` used in BOLT 11.
-Even if `node_id` is public, we force the use of blinding paths to keep these features.
+Even if `offer_node_id` is public, we force the use of blinding paths to keep these features.
 If the recipient does not care about the added privacy offered by blinded paths, they can create a path of length 1 with only themselves.
 
 Rather than provide detailed per-hop-payinfo for each hop in a blinded path, we aggregate the fees and CLTV deltas.
 This avoids trivially revealing any distinguishing non-uniformity which may distinguish the path.
 
-It's often useful to provide capacity hints, particularly where more
-than one blinded path is included, for payers to use multi-part
-payments.
+The invoice issuer is allowed to ignore `invoice_request_payer_note` (it has an odd
+number, so is optional), but must still copy it exactly.
```

Comment:
This is the rationale section, the requirements are clear about this:

```
  - if creating an `invoice` for an offer with `offer_send_invoice`:
     - MUST copy all fields from the offer (including unknown fields).
```
But perhaps ignore is too strong here; there's already a section on this in the "invoice_request" rationale.

```
`invoice_request_payer_note` allows you to compliment, taunt, or otherwise engrave
graffiti into the invoice for all to see.
```

I'll remove this.